samples/caf_sensor_manager/README.rst

```diff
@@ -22,17 +22,14 @@ The sample supports the following development kits:
 Overview
 ********
 
-The sample:
+The sample provides simulated input data using :ref:`sensor_sim`.
+The input data simulates acceleration in X, Y, and Z axes, with the acceleration generated as value of a periodic wave signal.
+Whenever new data is generated, the sensor manager module generates the appropriate :c:struct:`sensor_event` instances that includes the generated data.
 
-1. Provides simulated input data using sensor_sim and :ref:`caf_sensor_manager`.
-#. Aggregates data using :ref:`caf_sensor_data_aggregator`.
-#. When aggregator buffer is full sends data to workload_sim module.
-#. Simulates work using workload_sim.
-
-Configuration
-*************
-
-|config|
+The data from the events is then passed to the :ref:`caf_sensor_data_aggregator`, which aggregates it.
```

Comment:
I'd remove `, which aggregates it`, as we have "aggregate" four times on 2 lines.